src/drift/Arguments/AuthType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace rangers.antidrift.drift.Arguments
+{
+    public enum AuthType
```

Comment:
I would add [interactive login](https://docs.microsoft.com/en-us/azure/devops/integrate/concepts/dotnet-client-libraries?view=azure-devops#pattern-for-use) option as well.

Reply:
Considering the nature of the application, does it really need to support interactive login?

Reply:
Absolutely yes to me. It's command line tool I can use manually. With interactive logon it would cache credentials like Visual Studio and that would significantly simplify the usage. We may also need something like bearer token support for use inside the pipeline task. I had no experience with writing pipeline tasks yet but as I understand there is a bearer token in the environment variable available at the agent which can be used by task to access Azure DevOps service.

Reply:
Resolved in the latest commit.